src/test/java/stepsDesign/TestGCS1.java

```diff
@@ -0,0 +1,70 @@
+package stepsDesign;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import java.io.IOException;
+import java.util.UUID;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import pages.actions.*;
+import pages.locators.CdfStudioLocators;
+import utils.SeleniumDriver;
+import utils.SeleniumHelper;
+
+import static utils.RemoteClass.createDriverFromSession;
+
+public class TestGCS1 {
+    static int i = 0;
+
+
+    @Given("Open Datafusion Project to configure pipeline1")
+    public void openDatafusionProjectToConfigurePipeline() throws IOException, InterruptedException {
+
+        RemoteWebDriver driver = createDriverFromSession(SeleniumDriver.session(), SeleniumDriver.url);
+            SeleniumDriver.openPage(SeleniumHelper.readParameters("CDFURL"));
+        SeleniumDriver.waitForPageToLoad();
+    }
+
+    @When("Source is GCS bucket1")
+    public void sourceIsGCSBucket() throws InterruptedException {
+        CdfStudioActions.selectGCS();
+
+    }
+
+
+    @When("Target is BigQuery1")
```

Comment:
Why do we need BigQuery1? Also try to move it into a common class. E.g. try using interface with default method, this way we can structure it nicely